src/System.Linq/samples/System.Linq.Samples.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{D8207664-0C98-4DC9-8249-02546D21C7A4}</ProjectGuid>
+    <OutputType>Library</OutputType>
```

Comment:
Unless the OutputType is set to "Exe", then there won't be an entry point method in the assembly, even if you have a "static Main()" method. If you want a runnable program to be produced from the project (including a runtime and any dependencies declared in the project.json file) then you will need a bit more elaborate of a setup than just setting the OutputType. Setting the output type will give you a .exe file, but that's about it. Check out buildtools to see how we are doing it there for some of our tools projects.